Earthfile

```diff
@@ -15,13 +15,16 @@ test:
     # Run unit tests
     RUN mix test
 
-    # Run installer tests
-    WORKDIR /src/installer
-    RUN mix test
+    IF [ "$RUN_INSTALLER_TESTS" = "1" ]
```

Comment:
@agbell hrm, this is not working, but it should according to the docs. Is this perhaps a Earthfile version issue?

Reply:
It must be, its a newer feature.  in the GitHub action, it has the download step, we could bump it to 
https://github.com/earthly/earthly/releases/tag/v0.5.16

Reply:
It works!!!